### DISCUSSION TITLE
Can anyone help me with this error of lab1?

### DISCUSSION THREAD

**Discussion by Josephood7** [bol-edu/HLS-SOC-Discussions#54] [General]
I'm trying to run this Python code on pynq but it fails.

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/52d21141-4cf7-4323-a120-efb0445c719e)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/f67b7029-c45f-4e34-a545-f2963664bcbb)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/0e17a98a-6676-4abe-8229-8a92faff28b2)

**Comment by jxes993409** [comment]
You have to upload the .bit and .hwh file to jupyter notebook, not the FPGA rent server.
For example, if I upload the file to this folder, its path is /home/xilinx/jupyter_notebooks/yourbitfile.bit.
![screenshot](https://github.com/bol-edu/HLS-SOC-Discussions/assets/68891412/2c39c19f-0222-4607-8d41-a1cddce93e0f)

**Reply by Josephood7** [reply]
Ah, that makes sense.
Thank you so much.
It works perfectly ~
